subsys/partition_manager/Kconfig

```diff
@@ -158,6 +159,18 @@ partition=TFM_EXTRA
 partition-size=0x10000
 rsource "Kconfig.template.partition_size"
 
+if NRF_MODEM_LIB
+
+config TFM_PROFILE
+	string "The build profile used for TFM Secure image."
+	depends on BUILD_WITH_TFM
```

Comment:
This dependence is not needed here, i guess the whole block is under BUILD_WITH_TFM